src/main/kotlin/org/rust/ide/newProject/ui/AddUserTemplateDialog.kt

```diff
@@ -43,22 +43,27 @@ class AddUserTemplateDialog : DialogWrapper(null) {
 
     override fun doOKAction() {
         // TODO: Find a better way to handle dialog form validation
-        if (nameField.text.isBlank()) return
-        if (RsUserTemplatesState.getInstance().templates.any { it.name == nameField.text }) return
+        val name = nameField.text.trim()
```

Comment:
nit: maybe introduce `trimmedText` extension property for text fields since we use it several times? We even have a file for such extensions - `src/main/kotlin/org/rust/openapiext/ui.kt`.
Don't insist